lib/diagnostic.js

```diff
@@ -42,6 +74,8 @@ db.dropDatabase();
 
 // your code begins here
 
+db.house.update({ name: "Arryn"}, { name: "Arryn"});
```

Comment:
This change the entire document to only have the `name` key.  `$unset` should be used to remove an individual key.

```javascript
db.houses.update({
  name: 'House Arryn'
}, {
  $unset: {
    motto: ''
  }
});
```